pkg/pprof/merge_test.go

```diff
@@ -33,51 +35,6 @@ func Test_Merge_Self(t *testing.T) {
 	testhelper.EqualProto(t, p.Profile, m.Profile())
 }
 
-func Test_Merge_ZeroReferences(t *testing.T) {
```

Comment:
I had to remove the test as it's no longer valid: invalid elements are discarded at normalization/merge